python/paddle/jit/sot/opcode_translator/executor/variables/__init__.py

```diff
@@ -27,6 +27,8 @@
     GlobalVariable,
     ModuleVariable,
     NullVariable,
+    NumpyArrVariable,
+    NumpyNumVariable,
```

Comment:
```suggestion
    NumpyArrayVariable,
    NumpyNumberVariable,
```

这里写全吧，这里不需要缩写

到了变量名时候可以适当缩写，`np_arr_var = NumpyArrayVariable()` 这种